docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md

```diff
@@ -26,11 +31,26 @@ as follows:
   devcon.exe update sgx_psw.inf "SWC\VEN_INT&DEV_0E0C"
   ```
 
+### Verifying PSW installation
 You can verify that the correct version of Intel SGX PSW is installed by using
 Windows Explorer to open `C:\Windows\System32`. You should be able to find
 file `sgx_urts.dll` if PSW is installed. Right click on `sgx_urts.dll`,
 choose `Properties` and then find `Product version` on the `Details` tab.
-The version should be "2.12.xxx.xxx" or above.
+The version should be `2.12.xxx.xxx` or above.
+
+To verify that Intel SGX PSW is running, use the following command:
+
+```cmd
+sc query aesmservice
+```
+
+The state of the service should be "running" (4). Follow Intel's documentation for
+troubleshooting. In case the AESM service was stopped for some reasons, restart it
+using the following command from Powershell.
+
+```powershell
+Start-Service "AESMService"
```

Comment:
Shouldn't this be Restart-Service or do Stop-Service first

Reply:
This was copied over from WindowsManualSGX1Prereqs.md and seems to be meant for starting a stopped AESMService. But let me rewrite it to be more clear.

Reply:
Edited for clarity.